assets/js/theme/common/carousel/utils/setTabindexes.js

```diff
@@ -0,0 +1,22 @@
+const allFocusableElementsSelector = '[href], button, input, textarea, select, details, [contenteditable="true"], [tabindex]';
+
+export default ($slides, $prevArrow, $nextArrow, actualSlide, actualSlideCount) => {
+    $slides.each((index, element) => {
+        const $element = $(element);
+        const tabIndex = $element.hasClass('slick-active') ? 0 : -1;
+        if (!$element.hasClass('js-product-slide')) {
+            $element.attr('tabindex', tabIndex);
+        }
+
+        $element.find(allFocusableElementsSelector).each((idx, child) => {
+            $(child).attr('tabindex', tabIndex);
+        });
+    });
+
+    if ($prevArrow.length === 0
+        || $nextArrow.length === 0
+        || $prevArrow.hasClass('js-hero-prev-arrow')) return;
```

Comment:
If we have no arrows - we do not need to work with them. If we have arrows of main page carousel - we also do not need to disable them, because this carousel has infinite loop